Packs/Azure-Enrichment-Remediation/Playbooks/Azure_-_Network_Security_Group_Remediation.yml

```diff
@@ -992,6 +1042,15 @@ tasks:
                       value:
                         simple: ${inputs.AzureVMPrivateIP}
```

Comment:
since this is a context value in the condition you don't need to point it with the form of `${<key>}`
You can just try the suggestion and see how it will paste that.